libazureinit/README.md

```diff
@@ -0,0 +1,15 @@
+# libazureinit
+
+A common library for provisioning Linux VMs on Azure.
+
+Features would include like:
+
+* minimal configurations of virtual machines
+* provisioning metadata
+* basic features for instance initialisation
+
+On the other hand, [azure-init](https://github.com/Azure/azure-init) is a reference implementation which actually calls API provided by libazureinit.
+
+Goal is to make the library provide common API to be used by other provisioning implementations.
```

Comment:
The goal is to provide APIs to for other components to perform VM provisioning on Azure platform.